cstar/roms/runtime_settings.py

```diff
@@ -0,0 +1,806 @@
+            )
+            lines.append(
+                f"- Horizontal Laplacian TKE mixing [`q2nu2`, m2/s] = {self.my_bak_mixing.q2nu2},"
+            )
+            lines.append(
+                f"- Horizontal biharmonic TKE mixing [`q2nu4`, m4/s] = {self.my_bak_mixing.q2nu4},"
+            )
+
+        if self.sss_correction is not None:
+            lines.append(
+                f"SSS correction (`ROMSRuntimeSettings.sss_correction`): {self.sss_correction}"
+            )
+        if self.sst_correction is not None:
+            lines.append(
+                f"SST correction (`ROMSRuntimeSettings.sst_correction`): {self.sst_correction}"
+            )
+        if self.ubind is not None:
+            lines.append(
+                f"Open boundary binding velocity (`ROMSRuntimeSettings.ubind`, m/s) = {self.ubind}"
+            )
+        if self.v_sponge is not None:
+            lines.append(
+                f"Maximum sponge layer viscosity (`ROMSRuntimeSettings.v_sponge`, m2/s) = {self.v_sponge}"
+            )
+        if self.climatology is not None:
+            lines.append(
+                f"Climatology data files (`ROMSRuntimeSettings.climatology`): {self.climatology}"
+            )
+
+        return "\n".join(lines)
+
+    def __repr__(self) -> str:
+        """Return a full debug-style string representation of the settings.
+
+        Returns
+        -------
+        str
+            A detailed summary of the object, suitable for debugging.
+        """
+        attrs = {
+            "title": self.title,
+            "time_stepping": dict(self.time_stepping),
+            "bottom_drag": dict(self.bottom_drag),
+            "initial": dict(self.initial),
+            "forcing": [str(f) for f in self.forcing],
+            "output_root_name": self.output_root_name,
+            "grid": str(self.grid) if self.grid else None,
+            "climatology": str(self.climatology) if self.climatology else None,
+            "s_coord": dict(self.s_coord) if self.s_coord else None,
+            "rho0": self.rho0,
+            "lin_rho_eos": dict(self.lin_rho_eos) if self.lin_rho_eos else None,
+            "marbl_biogeochemistry": dict(self.marbl_biogeochemistry)
+            if self.marbl_biogeochemistry
+            else None,
+            "lateral_visc": self.lateral_visc,
+            "gamma2": self.gamma2,
+            "tracer_diff2": self.tracer_diff2
+            if self.tracer_diff2 is not None
+            else None,
+            "vertical_mixing": dict(self.vertical_mixing)
+            if self.vertical_mixing
+            else None,
+            "my_bak_mixing": dict(self.my_bak_mixing) if self.my_bak_mixing else None,
+            "sss_correction": self.sss_correction,
+            "sst_correction": self.sst_correction,
+            "ubind": self.ubind,
+            "v_sponge": self.v_sponge,
+        }
+
+        inner = ", ".join(f"{k}={repr(v)}" for k, v in attrs.items() if v is not None)
```

Comment:
consider using the formatting shortcut `f"{foo=}` where possible to avoid having to synchronize the hardcoded keys above and the actual property names on the objects.